include/eve/module/real/core/function/regular/generic/load.hpp

```diff
@@ -120,6 +143,47 @@ namespace eve::detail
   {
     return static_cast<T>(*p);
   }
+
+#if defined(SPY_COMPILER_IS_CLANG) or defined(SPY_COMPILER_IS_GCC)
+#define DISABLE_SANITIZERS __attribute__((no_sanitize_address)) __attribute__((no_sanitize_thread))
+#elif defined(SPY_COMPILER_IS_MSVC)
+#define DISABLE_SANITIZERS __declspec(no_sanitize_address)
+#else
+#define DISABLE_SANITIZERS
+#endif
+
+  // making unsafe(load) call intinsics requires a lot of work and is not very portable
+  // due to what called functions it will affect.
+  // Since the unsafe is mostly used in corner cases, will do it scalar in asan mode.
+  template<typename Ptr, typename Cardinal>
+  DISABLE_SANITIZERS auto load_(EVE_SUPPORTS(cpu_), unsafe_type, Ptr ptr, Cardinal const& N) noexcept
+    requires sanitizers_are_on && (requires(Ptr ptr, Cardinal const& N) { eve::load(ptr, N); })
+  {
+    using e_t = std::remove_cvref_t<decltype(*ptr)>;
+    using r_t = as_wide_t< e_t, typename Cardinal::type >;
+
+    r_t that;
+
+    for (std::ptrdiff_t i = 0; i != N(); ++i) that.set(i, ptr[i]);
```

Comment:
This can be replaced by :

```
return []<std::size_t... I>( std::index_sequence<I...>)
{
  return r_t{ ptr[I]... };
}( std::make_index_sequence<N::value>{});
```

it is still filled in a scalar way BUT call the mm_set or similar intrinsic.
Also the loop is garanteed to be compile-time unrolled.

Reply:
Doens't work. Asan still complains at you for doing this and I don't believe i can easily put an asan attribute on a lambda. Let's not

Reply:
OK